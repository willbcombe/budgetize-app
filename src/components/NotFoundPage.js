import React from 'react';
import {BrowserRouter, Route, Link} from 'react-router-dom';

const NotFoundPage= () =>(
    <div>
        404! NOT FOUND - <Link to="/"> Go Home</Link>
    </div>
);
export default NotFoundPage;