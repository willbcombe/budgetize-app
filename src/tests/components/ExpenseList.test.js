import React from 'react';
import {shallow} from 'enzyme';
import {ExpenseList} from '../../components/ExpenseList';
import expense from '../fixtures/expense';

test('should render ExpenseList w/ expenses', () => {
    const wrapper = shallow(<ExpenseList expenses={expense}/>);
    expect(wrapper).toMatchSnapshot();
});


test('should render ExpenseList w/ expenses', () => {
    const wrapper = shallow(<ExpenseList expenses={[]}/>);
    expect(wrapper).toMatchSnapshot();
});