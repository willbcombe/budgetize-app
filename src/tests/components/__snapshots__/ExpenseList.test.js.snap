// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList w/ expenses 1`] = `
<div>
  <ExpenseListItem
    amount={195}
    createdAt={0}
    description="Gum"
    id="1"
    key="1"
  />
  <ExpenseListItem
    amount={109500}
    createdAt={-345600000}
    description="Rent"
    id="2"
    key="2"
  />
  <ExpenseListItem
    amount={4500}
    createdAt={345600000}
    description="Credit Card"
    id="3"
    key="3"
  />
</div>
`;

exports[`should render ExpenseList w/ expenses 2`] = `
<div>
  <p>
    No Expenses
  </p>
</div>
`;
